### PR TITLE
fix:The log flags are not set in apiserver using logs.Config.

### PR DESCRIPTION
**Pull request by ONE7live**
Signed-off-by: ONE7live <wangqi_yewu@cmss.chinamobile.com>

#237 

fix:The log flags are not set in apiserver using logs.Config.

### CODE REVIEW

File: cmd/apiserver/app/options/options.go [hunk 1]
```diff
@@ -31,6 +32,7 @@ type ClusterPediaServerOptions struct {
 	Admission      *genericoptions.AdmissionOptions
 	//      Traces         *genericoptions.TracingOptions
 
+	Logs    *logs.Options
```
Comment (Iceber) [inline cmd/apiserver/app/options/options.go:35]: `logs.Options`  looks better on top
```go
	SecureServing  *genericoptions.SecureServingOptionsWithLoopback
	Authentication *genericoptions.DelegatingAuthenticationOptions
	Authorization  *genericoptions.DelegatingAuthorizationOptions
	Audit          *genericoptions.AuditOptions
	Features       *genericoptions.FeatureOptions
	CoreAPI        *genericoptions.CoreAPIOptions
	FeatureGate    featuregate.FeatureGate
	Admission      *genericoptions.AdmissionOptions
	Logs               *logs.Options
```
 > It' s provided by kube

The usage position also needs to be changed.

**Review by Iceber — COMMENTED** [review]
others lgtm, Thank you for contributing.

File: cmd/apiserver/app/options/options.go (hunk 1 above)
Comment (ONE7live) [reply cmd/apiserver/app/options/options.go]: Thank you for your suggestion, I have made changes, please review

**Review by Iceber — APPROVED** [review]
LGTM

**Pull request merged**